frame/assets/src/benchmarking.rs

```diff
@@ -514,8 +514,7 @@ benchmarks_instance_pallet! {
 			SystemOrigin::Signed(new_account.clone()).into(),
 			asset_id
 		).is_ok());
-		// `touch` should reserve some balance of the caller...
-		assert!(!T::Currency::reserved_balance(&new_account).is_zero());
+		assert_eq!(T::Currency::reserved_balance(&new_account), T::AssetAccountDeposit::get());
```

Comment:
Maybe you can re-add the comment? Basically this pr doesn't "fix" anything, it just ensures that the right amount is reserved or?

Reply:
comment re-added,
yeah, PR fixes just benchmark and right amount comparison, which didnt work for `AssetAccountDeposit = 0`